### PR TITLE
Loaders: Replace JSZip and gunzip with fflate.

### LINKED ISSUE #20941: [124] TiltLoader breaks in a module env
Repro: https://codesandbox.io/s/elegant-villani-9d3hr?file=/src/index.js

```jsx
import { TiltLoader } from "three/examples/jsm/loaders/TiltLoader"

new TiltLoader().load("/model.tilt", (scene) => console.log(scene))
```

This is the error message in a local env (Parcel):

```text
TiltLoader.js:42 TypeError: _jszipModuleMin.JSZip is not a constructor
    at TiltLoader.parse (TiltLoader.js:59)
    at Object.onLoad (TiltLoader.js:32)
    at XMLHttpRequest.<anonymous> (three.module.js:36749)
```

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: Fixed #20941.

**Description**

This PR replaces the loader dependencies JSZip and gunzip with fflate.

To completely remove JSZip, it would be necessary to refactor some code in the editor (although I don't think this task is that urgent and can be done later).

It was also necessary to slightly enhance `modularize.js` so it's possible to convert the usage of fflate from js -> jsm.

### CODE REVIEW

File: examples/js/loaders/3MFLoader.js [hunk 1]
```diff
@@ -133,15 +133,15 @@ THREE.ThreeMFLoader.prototype = Object.assign( Object.create( THREE.Loader.proto
 
 			//
 
-			var relsView = new Uint8Array( zip.file( relsName ).asArrayBuffer() );
+			var relsView = new Uint8Array( zip[ relsName ].buffer );
```
Comment (101arrowz) [inline examples/js/loaders/3MFLoader.js:136]: This is the same as:
```
var relsView = zip[ relsName ];
```

File: examples/js/loaders/3MFLoader.js [hunk 2]
```diff
@@ -133,15 +133,15 @@ THREE.ThreeMFLoader.prototype = Object.assign( Object.create( THREE.Loader.proto
...
 			var relsFileText = THREE.LoaderUtils.decodeText( relsView );
 			rels = parseRelsXml( relsFileText );
 
 			//
 
 			if ( modelRelsName ) {
 
-				var relsView = new Uint8Array( zip.file( modelRelsName ).asArrayBuffer() );
+				var relsView = new Uint8Array( zip[ modelRelsName ].buffer );
```
Comment (101arrowz) [inline examples/js/loaders/3MFLoader.js:144]: See above. `new Uint8Array(u8Array.buffer)` is the same as `u8Array`.

File: examples/js/loaders/3MFLoader.js [hunk 3]
```diff
@@ -152,7 +152,7 @@ THREE.ThreeMFLoader.prototype = Object.assign( Object.create( THREE.Loader.proto
 			for ( var i = 0; i < modelPartNames.length; i ++ ) {
 
 				var modelPart = modelPartNames[ i ];
-				var view = new Uint8Array( zip.file( modelPart ).asArrayBuffer() );
+				var view = new Uint8Array( zip[ modelPart ].buffer );
```
Comment (101arrowz) [inline examples/js/loaders/3MFLoader.js:155]: See above

File: examples/js/loaders/AMFLoader.js [hunk 4]
```diff
@@ -101,7 +101,7 @@ THREE.AMFLoader.prototype = Object.assign( Object.create( THREE.Loader.prototype
 				}
 
 				console.log( 'THREE.AMFLoader: Trying to load file asset: ' + file );
-				view = new DataView( zip.file( file ).asArrayBuffer() );
+				view = new DataView( zip[ file ].buffer );
```
Comment (101arrowz) [inline examples/js/loaders/AMFLoader.js:104]: ...

**Review by 101arrowz — COMMENTED** [review]
Looks mostly correct, and the demos that I tested do seem to work.

File: examples/js/loaders/AMFLoader.js (hunk 4 above)
Comment (Mugen87) [reply examples/js/loaders/AMFLoader.js:104]: If I remove `.buffer` here I get:

> AMFLoader.js:65 TypeError: First argument to DataView constructor must be an ArrayBuffer

Comment (101arrowz) [reply examples/js/loaders/AMFLoader.js:104]: My bad, that was a mistake.

**Comment by Mugen87** [comment]
Okay, I've cleaned up the usage of fflate in `3MFLoader`.

**Review by 101arrowz — APPROVED** [review]

**Comment by mrdoob** [comment]
This is awesome. You guys are the best! 🤗

**Comment by mrdoob** [comment]
Thanks! ✨

File: editor/sw.js [hunk 5]
```diff
@@ -13,6 +13,7 @@ const assets = [
 	'../examples/jsm/controls/TransformControls.js',
 
 	'../examples/jsm/libs/chevrotain.module.min.js',
+	'../examples/jsm/libs/fflate.module.min.js',
 	'../examples/jsm/libs/inflate.module.min.js',
 	'../examples/jsm/libs/jszip.module.min.js',
```
Comment (mrdoob) [inline editor/sw.js:18]: Do we still need these two?

Comment (mrdoob) [reply editor/sw.js:18]: Ah, I just saw https://github.com/mrdoob/three.js/issues/20941#issuecomment-752032414

**Pull request merged**